src/swe/swe_ceed_impl.h

```diff
@@ -125,7 +48,7 @@ CEED_QFUNCTION_HELPER void SWERiemannFlux_Roe(const CeedScalar gravity, const Ce
 #pragma clang diagnostic ignored "-Wvla"
 
 // SWE interior flux operator Q-function
-CEED_QFUNCTION(SWEFlux_Roe)(void *ctx, CeedInt Q, const CeedScalar *const in[], CeedScalar *const out[]) {
+CEED_QFUNCTION(SWEFlux)(void *ctx, CeedInt Q, const CeedScalar *const in[], CeedScalar *const out[], RiemannFluxType flux_type) {
```

Comment:
This is called by a qfunction, but does not implement the qfunction interface (it has an extra argument).
```suggestion
CEED_QFUNCTION_HELPER int SWEFlux(void *ctx, CeedInt Q, const CeedScalar *const in[], CeedScalar *const out[], RiemannFluxType flux_type) {
```